code/API_definitions/number-verification.yaml

```
@@ -2,56 +2,76 @@ openapi: 3.0.3
 info:
   title: Number Verification
   description: |
-    Service API to verify that the provided **mobile phone number** is the one used in the device. It verifies that the user is using a device with the same *mobile phone number* as it is declared.
-    It also makes it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token.
+    This API can verify or retrieve the **mobile phone number** that is currently allocated by the network operator to the SIM in the end user's device
 
-    In this API **phone number** term refers to the mobile phone number.
+    In this API **phone number** refers to the mobile phone number.
 
     # Introduction
 
-    Number Verification API performs real-time checks to verify the phone number of the mobile device being used to access a service provider (SP) service, where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
-
-    It uses direct mobile network connections to verify possession of a phone number in the background without requiring user interaction. There are neither OTPs (One-time passwords) received by SMS nor authenticator app downloads, so it is much simpler. It can be used at sign up, login, or transaction time to validate that a user's SIM (Subscriber Identity Module) is both actively connected to the mobile network and not spoofed or cloned.
+    Number Verification API performs real-time checks to verify the phone number of the mobile device being used to access a service provider (SP) service, either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
```

change "a service provider (SP) service" --> "the application" ?

can we clarify more this paragraph ? " it is used", "they" ?? not sure what it is can we use application focused text ?
```suggestion
The Number Verification API is used by the API consumer to perform real-time checks to verify the phone number of a mobile device being used to access the application. This check can be done either by the network, returning "true" or false, or by the application, by matching the phone number returned by the network with the phone number of the device that is being used.```